src/style.css

```diff
@@ -2,0 +1,50 @@
+body {
+  margin: 0;
+}
+
+main {
+  width: 96px;
+  height: 116px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url('./images/star.svg');
```

Comment:
The selectors '.stars--0 .stars__star:nth-child(-n + 0)' will not select any element as it's trying to select up to the 0th child, which doesn't exist. You might want to remove this rule if it's not serving any purpose.